data/dataloader.py

```diff
@@ -87,7 +90,55 @@ def to_cuda(item, computing_device, label=False):
     else:
         item = Variable(item.to(computing_device)).float()
     return item
+    
 
+def create_class_dict (lbs_all_classes):
```

Comment:
Let's add function adders for code maintainability. Right now, I have no idea what this function does and what the input argument means. Will be helpful for me to have this info at the top before I review this function

Reply:
Following previous function commenting style, the functions' descriptions are under the functions' names. But yea, unlike major function, these minor functions has no description on the input arguments.

Reply:
Just take a look of my function descriptions. They are not very clear and detailed. You are right, lets add elaborate function headers

Reply:
Added more descriptions. Let me know which are still unclear.